### PR TITLE
Adds docs for keep-alive #2334

### PR DESCRIPTION
**Pull request by benarent**
Adds the defaults and keep alive settings into admin-guide teleport.yaml.

### CODE REVIEW

File: docs/3.2/admin-guide.md [hunk 1]
```diff
@@ -368,6 +368,13 @@ auth_service:
     # certificates expire in the middle of an active SSH session. (default is 'no')
     disconnect_expired_cert: no
 
+    # Determines the interval at which Teleport will send keep-alive messages. The 
+    # default value mirrors sshd at 15 minutes.  keep_alive_count_max is the number 
+    # of messed keep-alive messages before the server tears down the connection to the
```
Comment (klizhentas) [inline docs/3.2/admin-guide.md]: `messed` -> `missed`

File: docs/3.2/admin-guide.md [hunk 2]
```diff
@@ -368,6 +368,13 @@ auth_service:
     # certificates expire in the middle of an active SSH session. (default is 'no')
     disconnect_expired_cert: no
 
+    # Determines the interval at which Teleport will send keep-alive messages. The 
+    # default value mirrors sshd at 15 minutes.  keep_alive_count_max is the number 
+    # of messed keep-alive messages before the server tears down the connection to the 
+    # client. The default value mirrors sshd at 3
```
Comment (klizhentas) [inline docs/3.2/admin-guide.md]: Remove `he default value mirrors sshd at 3`

**Review by klizhentas — CHANGES_REQUESTED** [review]

**Comment by webvictim** [comment]
(same changes to both files)

**Comment by klizhentas** [comment]
@benarent ping

**Comment by benarent** [comment]
Thanks for the ping @klizhentas PTAL

**Pull request merged**